### PR TITLE
Choose tokens

### PR DESCRIPTION
**Pull request by rcoderdev**

### CODE REVIEW

File: frontend/src/chain/utils/hasMultipleCoinsSupport.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Chain } from '../../model/chain';
```
Comment (enriquesouza) [inline frontend/src/chain/utils/hasMultipleCoinsSupport.ts:1]: We must avoid this kind of thing. The service must handle the coins behaviors. 

CoinService.hasTokens OR 

something like TokenStore.filter(chain... && isNativeToken == false).length > 0. So, if it has tokens, then you can show it.

Therefore, it must be dynamic and not hardcoded.

**Review by enriquesouza — CHANGES_REQUESTED** [review]
Please, do not hardcode any chain behavior. Read my comments.

File: frontend/src/chain/utils/hasMultipleCoinsSupport.ts (hunk 1 above)
Comment (rcoderdev) [reply frontend/src/chain/utils/hasMultipleCoinsSupport.ts]: ✅ https://github.com/vultisig/vultisig-windows/pull/157/commits/63d516b946627eb9685aa5c32e047532652aee8d

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**